apps/drec-ui/package.json

```diff
@@ -0,0 +1,105 @@
+{
+    "name": "@energyweb/origin-drec-ui",
+    "version": "0.0.1",
+    "description": "",
+    "private": true,
+    "author": "EnergyWeb DevHub GmbH; Florin Georgescu, florin.georgescu@energyweb.org",
+    "license": "GPL-3.0-or-later",
+    "scripts": {
+        "start:dev": "react-scripts start",
+        "build": "yarn config-env && react-scripts build",
```

Comment:
These command should also be adjusted if you are moving to `dev-env.sh` and `prod-start.sh`